src/pages/namsunkim/Feed/Components/AddComment/AddComment.scss

```
@@ -0,0 +1,32 @@
+
+@import '../../../styles/variable.scss';
+
+.addComment {
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+  height: 3.125rem;
+  padding: 0 0.9375rem;
+  border-top: 1px solid $border;
+
+  svg {
+    width: 1.5rem;
+    height: 1.5rem;
+  }
+
+  input {
+    width: 85%;
+    border: 0;
+  }
+
+  .enabledBtn {
+    background-color: $white;
+    color: $enabled;
+    border: 0;
+  }
+  .disabledBtn {
```

다른 속성간에는 한줄씩 공백을 주시는게 가독성 측면에서 좋습니다.
```suggestion

  .disabledBtn {
```

---

넵 수정하겠습니다!